### ISSUE TITLE
FlexibleColumnLayout: Opening of the column layout is not announced

### ISSUE THREAD

**Issue by pppwr7** [SAP/ui5-webcomponents-react#5889]
### Describe the bug

Hi Colleagus, As a part of accessibility testing, we received an issue that whenever a new column in the layout is made visible or hidden, the user is not announced regarding the same.

### Isolated Example

https://sap.github.io/ui5-webcomponents-react/?path=/story/layouts-floorplans-flexiblecolumnlayout--with-different-columns

### Reproduction steps

1. go to https://sap.github.io/ui5-webcomponents-react/?path=/story/layouts-floorplans-flexiblecolumnlayout--with-different-columns
2. click on any row in the table
3. New column of the layout is made visible but it is not announced.

### Expected Behaviour

User should be made aware of the changes in the UI

### UI5 Web Components for React Version

latest

### UI5 Web Components Version

latest

### Browser

Chrome

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @pppwr7 

Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.

___

Hi colleagues, 
the described behavior can also be observed in your example: https://sap.github.io/ui5-webcomponents/components/fiori/FlexibleColumnLayout/
